.github/workflows/atlantis-image.yml

```diff
@@ -158,3 +158,48 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
       - run: 'echo "No build required"'
+
+  test:
```

Comment:
Can we move this above the skip build job?

Reply:
Absolutely - done! :)